skbio/diversity/alpha/_pd.py

```diff
@@ -0,0 +1,410 @@
+    :math:`PD` is the sum of lengths (:math:`l`) of branches in both sets.
+
+    It is equivalent to ``pd(..., rooted=True, weight=False)``.
+
+    A variant of PD, which does not include the root in the calculation, was
+    referred to by some authors as unrooted phylogenetic diversity (uPD) [2]_,
+    as in contrast to rooted phylogenetic diversity (rPD, i.e., Faith's PD).
+    uPD is defined as:
+
+    .. math::
+
+       PD = \sum_{b \in T} l(b)
+
+    It is equivalent to ``pd(..., rooted=False, weight=False)``.
+
+    See ``faith_pd`` for a discussion of the root.
+
+    PD (rooted or unrooted) considers only the presence of taxa. Therefore, it
+    can be considered as the phylogenetic generalization of species richness.
+    However, there are advantages of incorporating abundance information in the
+    measurement [3]_. A generalized framework of abundance-weighted PD is
+    provided in [4]_.
+
+    Abundance-weighted rooted PD (equivalent to :math:`RBWPD_{1}` described in
+    [4]_) is analogous to the :math:`PD_{aw}` metric originally described in
+    [5]_ with a multiplier. It is defined as:
+
+    .. math::
+
+       PD = \sum_{b \in T \sqcup R} l(b) p(b)
+
+    where :math:`p` is the sum of relative (proportional) abundances of taxa
+    descending from branch (:math:`b`).
+
+    It is equivalent to ``pd(..., rooted=True, weight=True)``.
+
+    Abundance-weighted unrooted PD (equivalent to :math:`BWPD_{1}` described in
+    [4]_) is analogous to the :math:`\delta nPD` metric originally described in
+    [6]_ with a multiplier. It is defined as:
+
+    .. math::
+
+       PD = 2 \sum_{b \in T} l(b) \min(p(b),1-p(b))
+
+    In which the term :math:`2\min(p(b),1-p(b))` is the lesser of the relative
+    abundance of descending taxa on either side of a branch, multiplied by two.
+    It is referred to as the "balance" of taxon abundance in [4]_.
+
+    It is equivalent to ``pd(..., rooted=False, weight=True)``.
+
+    The contribution of taxon abundance to the metric can be adjusted using the
+    ``weighted`` parameter when it is a float within [0, 1]. This factor was
+    referred to as :math:`\theta` in [4]_. The metric, :math:`BWPD_{\theta}`,
+    referred to as the balance-weighted phylogenetic diversity in [4]_, is
+    defined as:
+
+    .. math::
+
+       PD = \sum_{b \in T} l(b) (2\min(p(b),1-p(b)))^\theta
+
+    It is equivalent to ``pd(..., rooted=False, weight=theta)``.
+
+    This metric falls back to unweighted PD when :math:`\theta=0` or fully-
+    weighted PD when :math:`\theta=1`. The original publication tested
+    :math:`\theta=0.25` or :math:`0.5` [4]_.
+
+    The parameter :math:`\theta` is analogous to the parameter :math:`\alpha`
+    in the generalized UniFrac metric [7]_.
+
+    Likewise, the rooted version of balance-weighted phylogenetic diversity,
+    :math:`RBWPD_{\theta}` [4]_ (although "balance" is not involved), is
+    defined as:
+
+        PD = \sum_{b \in T \sqcup R} l(b) p(b)^\theta
```

Comment:
@qiyunzhu @mortonjt Apologies for the late catch, but I built the docs locally and line 348 does not render correctly.